zone/attack.cpp

```diff
@@ -1907,12 +1907,12 @@ bool Client::Death(Mob* killerMob, int64 damage, uint16 spell, EQ::skills::Skill
 
 	if ((GetLevel() < RuleI(Character, DeathExpLossLevel)) || (GetLevel() > RuleI(Character, DeathExpLossMaxLevel)) || IsBecomeNPC()) {
 		exploss = 0;
-	} else if (killerMob) {
-		if (killerMob->IsClient()) {
+	} else if (killer_mob) {
+		if (killer_mob->IsClient()) {
 			exploss = 0;
-		} else if (killerMob->GetOwner() && killerMob->GetOwner()->IsClient()) {
+		} else if (killer_mob->GetOwner() && killer_mob->GetOwner()->IsClient()) {
 			exploss = 0;
-		} else if (killerMob->IsBot()) {
+		} else if (killer_mob->IsBot()) {
 			exploss = 0;
 		}
```

Comment:
Could maybe consolidate this into one condition like this.
```cpp
Could maybe consolidate this into one condition like this.
```cpp
if (
	killer_mob->IsOfClientBot() ||
	(killer_mob->GetOwner() && killer_mob->GetOwner()->IsOfClientBot())
)```